packages/vscode-extension/src/plugins/network/network-plugin.ts

```diff
@@ -56,182 +49,45 @@ async function initialize() {
 export class NetworkPlugin implements ToolPlugin {
   public readonly id: ToolKey = NETWORK_PLUGIN_ID;
   public readonly label = "Network";
-  public readonly persist = true;
-
-  public pluginAvailable = true;
+  public readonly persist = false;
   public toolInstalled = false;
 
-  private devtoolsListeners: Disposable[] = [];
-  private broadcastListeners: BroadcastListener[] = [];
+  private readonly strategy: InspectorStrategy;
 
-  constructor(private readonly inspectorBridge: RadonInspectorBridge) {
+  constructor(
+    readonly inspectorBridge: RadonInspectorBridge,
+    readonly networkBridge: NetworkBridge
+  ) {
+    this.strategy = ENABLE_NEW_INSPECTOR
+      ? new NewInspectorStrategy(this)
+      : new LegacyInspectorStrategy(this);
```

Comment:
I wouldn't really call it "The Strategy Pattern" when you delegate literally every call to the object...
I'd just call the interface `NetworkInspector`, with two implementations.